File: 백준/Bronze/4153. 직각삼각형/직각삼각형.java
import java.io.*;
import java.util.*;

public class Main {
	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		StringBuilder sb = new StringBuilder();
		StringTokenizer st = new StringTokenizer(br.readLine());

		int x = Integer.parseInt(st.nextToken());
		int y = Integer.parseInt(st.nextToken());
		int z = Integer.parseInt(st.nextToken());
		
		while (x + y + z != 0) {
			if(Math.pow(Math.max(x, Math.max(y, z)), 2) * 2 == Math.pow(x, 2) + Math.pow(y, 2) + Math.pow(z, 2)) {
				sb.append("right\n");
			} else {
				sb.append("wrong\n");
			}
			
			st = new StringTokenizer(br.readLine());
			x = Integer.parseInt(st.nextToken());
			y = Integer.parseInt(st.nextToken());
			z = Integer.parseInt(st.nextToken());
		}
		
		System.out.println(sb);
		br.close();
	}
}

File: 백준/Bronze/4153. 직각삼각형/직각삼각형.py
tri = input().split()

while tri[0] != "0" or tri[1] != "0" or tri[2] != "0" :

    for i in range(3) :
        tri[i] = int(tri[i])

    if ((max(tri) ** 2) * 2) == ((tri[0] ** 2) + (tri[1] ** 2) + (tri[2] ** 2)) :
       print("right")
    else :
       print("wrong")

    tri = input().split()